pcdsdevices/rtds_ebd.py

```diff
@@ -44,6 +46,27 @@ def _do_move(self, state):
             self.out_cmd.put(1)
 
 
+class RTDSX0ThreeStage(BaseInterface, LightpathMixin):
+    """Three stages X,Y,Z, for solid drilling experiments"""
+
+    mmsx = Cpt(BeckhoffAxis, ':MMS:X', kind='normal')
+    mmsy = Cpt(BeckhoffAxis, ':MMS:Y', kind='normal')
+    mmsz = Cpt(BeckhoffAxis, ':MMS:Z', kind='normal')
```

Comment:
This is a unique component naming compared to other devices in the repo. That doesn't make it wrong, but it's worth thinking on it for a moment. The component names here are acronyms that nominally stand for "motion motor stepper x", for example.

Reply:
right, I based this nominally off of the naming convention in RTDSBase, where the pneumatic cylinders are labeled by their CCC; mpa1, mpa2, etc. That said, reviewing the more standard naming convention across other device types something like x_motor, y_motor, z_motor would probably be more in alignment with standard practice. I'll leave this for now as I wouldn't want to change the name without testing again, which would require waiting for another opportunity. I'll try to pay closer attention to package-wide naming standards moving forward.

Reply:
matching the other rtds devices is a good enough reason in my book